docs/detections/api/rules/signals-api-overview.asciidoc

```diff
@@ -353,4 +353,86 @@ Example response:
         }
     ]
 }
---------------------------------------------------
\ No newline at end of file
+--------------------------------------------------
+
+==== Assign or unassign users from alerts
+
+Allows you to assign and unassign users from alerts. 
+
+===== Request URL
+
+`POST <kibana host>:<port>/api/detection_engine/signals/assignees`
+
+===== Request body
+
+A JSON object with either a `assignees` and `ids` field:
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+
+|`assignees` |Object[] a|An array of unique identifiers (UIDs) for user profiles. Properties of the assignees object:
+
+* add: (Required, string[]) An array of assignees you want to add.
```

Comment:
```suggestion
* `add`: (Required, string[]) An array of assignees you want to add.
```

Reply:
So I styled this match the previous section (tags), which has been un-styled text for a few releases now. I can't find formal style guidelines on how we're supposed to style property tags, but I know we apply monospace to property and field names elsewhere in the API docs so I can update this and the tags section for 8.12 if it's needed.

Reply:
I think updating the formatting to monospace for 8.12 is good enough, probably not worth the effort of backporting to whenever the tags section was introduced.
We have [this page](https://docs.elastic.dev/content-design/emphasis) in the Content Design guidelines that mentions monospace formatting for parameter names.

Reply:
Agree! I did see that page and the style guidelines for parameters, but I was looking for guidelines on styling property names specifically. I think we use parameters, properties, and fields interchangeably in the API docs though, so the guidelines for one likely applies to the others. Down the line, it'd be good to make our terminology more consistent throughout the API docs 😄